_data/projects/clio.yml

```diff
@@ -0,0 +1,12 @@
+name: Clio
+desc: Clio is a pure functional lazy-evaluated programming language targeting decentralized and distributed systems.
+site: https://clio-lang.org
+tags:
+  - language
+  - programming
+  - distributed
+  - functional
+  - fp
```

Comment:
@garritfra Do you have any programming languages to add as tag?  It will be better

Reply:
Added with f045492